index.html

```diff
@@ -85,6 +86,176 @@ <h1>Hey There. I’m Akhtar</h1>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+        <div class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </div>
+    </section>
+    <div class="about-me-main">
+        <h2 class="about-me-title">About me</h2>
+        <p class="about-me-text">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#" class="get-resume">Get My Resume</a>
+        <hr class="ruler2" />
+        <div class="bullet-title doubledown">
+            <img class="diamond" src="diamond.png" alt="Bullet point" />
+            <h2 class="list-title">Language</h2>

```

Comment:
- [ ] **Please ensure that he technology categories ("Languages", "Frameworks" and "Skills") are grouped together in a ul, and each of them in is a `li`.**

I can see that the the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`. Great Work. However, the categories themselves are a part of list. Hence, they should be grouped in a `ul` tag. I have explained the benefit of using `ul` and `li` tags above.

Reply:
The title of the list also has been added in the list tags as per your requests. Thank you for the review.

Reply:
- [ ] Great work with this section, @Ak0706. But it doesn't look as the previous TSE described. You should have a nested list here.
Please check the example structure.

Example structure:
```html
<sectionn>
<ul>
  <li>Language
    <ul>
      <li>JavaScript</li>
      <li>Ruby</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
</li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
</section>
```

Reply:
Thank you for the feedback. I have now modified the list tag and made it a nested list tag.